charts/prefect-server/values.yaml

```diff
@@ -39,6 +39,9 @@ server:
   # -- sets PREFECT_UI_API_URL; should be publicly accessible API URL
   publicApiUrl: http://localhost:4200/api
 
+  # -- sets PREFECT_UI_URL
+  publicUiUrl: "http://localhost:4200"
```

Comment:
should we not set a default value, so that the behavior falls back to the underlying prefect library's behavior? genuinely not sure